### PR TITLE
feat(editor): Canvas chat UI & UX improvements

### PR DESCRIPTION
**Pull request by OlegIvaniv**
## Summary
This PR introduces several improvements to the chat interface and workflow execution state handling

### Chat UI Enhancements
- Adjusted font sizes in the chat to better match the visual hierarchy
- Reversed the color scheme of the chat button:
  - Now displays as filled-in red when closed
  - Changes to secondary color when open
- Updated chat button text to "Hide chat" when the panel is open
- Added a close button (x) to the right side of the chat panel, visible when the logs panel is not
- Improved scroll behavior to automatically scroll to the bottom of the chat panel when new messages arrive
- Fix `allowedFilesMimeTypes` property 

### Visual Improvements
- Softened the background color of disabled node parameters 

### Workflow Execution and Loading State
- Fixed the loading state to properly tie into the global workflow execution state
- Enhanced chat logs to stream individual executions rather than waiting for the entire workflow to finish
## Related Linear tickets, Github issues, and Community forum posts

https://github.com/user-attachments/assets/9bd0eed1-ef96-40d8-b15b-db9062706a40

## Review / Merge checklist

- [ ] PR title and summary are descriptive. ([conventions](../blob/master/.github/pull_request_title_conventions.md)) 
- [ ] [Docs updated](https://github.com/n8n-io/n8n-docs) or follow-up ticket created.
- [ ] Tests included. 
- [ ] PR Labeled with `release/backport` (if the PR is an urgent fix that needs to be backported)

### CODE REVIEW

File: packages/@n8n/chat/src/css/markdown.scss [hunk 1]
```diff
@@ -641,4 +640,15 @@ body {
 	body > a:first-child:focus {
 		top: 1rem;
 	}
+
+	// Lists
+	ul,
+	ol {
+		padding-left: 1.5rem;
```
Comment (MiloradFilipovic) [inline packages/@n8n/chat/src/css/markdown.scss:647]: Nit: Can we use spacing tokens where possible?

File: packages/editor-ui/src/components/CanvasChat/CanvasChat.vue [hunk 2]
```diff
@@ -173,15 +172,50 @@ const closePanel = () => {
...
-	return response;
+	// Watch for changes in the workflow execution status
+	const stopWatch = watch(
+		() => workflowsStore.getWorkflowExecution?.status,
+		(newStatus) => {
+			// If the status is no longer 'running', resolve the promise
+			if (newStatus && newStatus !== 'running') {
+				resolvePromise();
+				// Stop the watcher when the promise is resolved
+				stopWatch();
```
Comment (MiloradFilipovic) [inline packages/editor-ui/src/components/CanvasChat/CanvasChat.vue:191]: Nice!

**Review by MiloradFilipovic — DISMISSED** [review]
Nice! I left one really small nit in the comments and have two more visual things to note:
1. Is the chat button meant to be transparent when chat is open? It looks confusing to me
image
2. Execution status notice that appears on chat hover seems out of place to me:
execution_notice

**Comment by MiloradFilipovic** [comment]
@OlegIvaniv unrelated to this PR: How hard would it be to stream back these steps when agent is used with different trigger (webhook, telegram, ...)?

File: packages/@n8n/chat/src/css/markdown.scss (hunk 1 above)
Comment (OlegIvaniv) [reply packages/@n8n/chat/src/css/markdown.scss:647]: We don't have the standard spacing tokens from n8n here :/ Because chat-sdk is a standalone package.

**Comment by OlegIvaniv** [comment]
> Nice! I left one really small nit in the comments and have two more visual things to note:
> 
> 1. Is the chat button meant to be transparent when chat is open? It looks confusing to me
>    image
> 2. Execution status notice that appears on chat hover seems out of place to me:
>    execution_notice
>          
>            [
>              
>            
>                execution_notice
>              ](https://private-user-images.githubusercontent.com/2598782/391034888-893db6fb-bd4e-4172-81ff-2193bdbece65.gif?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzI4NzI4MzksIm5iZiI6MTczMjg3MjUzOSwicGF0aCI6Ii8yNTk4NzgyLzM5MTAzNDg4OC04OTNkYjZmYi1iZDRlLTQxNzItODFmZi0yMTkzYmRiZWNlNjUuZ2lmP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTEyOSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDExMjlUMDkyODU5WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MjZjZWNiNTQwNWRlY2M0MTI1MDQxNjNiZTBlOGJlYTM2N2ZlZDE1OWFkYTAwYjc5YzZiY2I2YjU3MWQ5ZjIwNiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.aKeHJ_bZhrADOt3JIveg_fkl8EmK5qWPc082oMTcERE)
>            
>            
>              
>                
>                  
>                
>                
>                  
>                  
>                
>              
>              [
>                
>                  
>                
>              ](https://private-user-images.githubusercontent.com/2598782/391034888-893db6fb-bd4e-4172-81ff-2193bdbece65.gif?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzI4NzI4MzksIm5iZiI6MTczMjg3MjUzOSwicGF0aCI6Ii8yNTk4NzgyLzM5MTAzNDg4OC04OTNkYjZmYi1iZDRlLTQxNzItODFmZi0yMTkzYmRiZWNlNjUuZ2lmP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTEyOSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDExMjlUMDkyODU5WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MjZjZWNiNTQwNWRlY2M0MTI1MDQxNjNiZTBlOGJlYTM2N2ZlZDE1OWFkYTAwYjc5YzZiY2I2YjU3MWQ5ZjIwNiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.aKeHJ_bZhrADOt3JIveg_fkl8EmK5qWPc082oMTcERE)

Thanks! Both should be addressed. Can you please re-review?

**Comment by OlegIvaniv** [comment]
> @OlegIvaniv unrelated to this PR: How hard would it be to stream back these steps when agent is used with different trigger (webhook, telegram, ...)?

Here we're relying on the websocket push events that we get in the editor like "afterNodeExecuted" so if we'd like to stream those, it should be doable by reusing some of the same logic

**Review by MiloradFilipovic — APPROVED** [review]
Thanks for addressing! I would love to see just a bit more of left padding for that notice icon so it's not so flush with text edge but feel free to ignore 🙌🏻

**Comment by janober** [comment]
Got released with [`n8n@1.71.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.71.0)

**Pull request merged**